Externals.cfg

```diff
@@ -1,9 +1,9 @@
 [regional_workflow]
 protocol = git
-repo_url = https://github.com/NOAA-EMC/regional_workflow
+repo_url = https://github.com/mkavulich/regional_workflow
```

Comment:
I'm guessing you're going to change this to the authoritative repo?

Reply:
Yes, the hash and repo have been updated, and the final Cheyenne test completed successfully. Thanks for the review, merging now!